### PR TITLE
fix aptos user uid

### PR DESCRIPTION
**Pull request by jachen-sh**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends curl ca-certificates procps && \
     rm -rf /var/lib/apt/lists/*
 
-RUN adduser --uid 1000 --home /aptos --gecos '' --disabled-password aptos
+RUN adduser --uid 1002 --home /aptos --gecos '' --disabled-password aptos
```
Comment (ivanpeng-sh) [inline Dockerfile:27]: Just curious, why the change in UID? Does aptos require the UID to be 1002? Or is 1000 overriding something else?

Comment (jachen-sh) [reply Dockerfile:27]: Want to keep them separate from other designations:
https://github.com/shinamicorp/tf-infra/blob/main/modules/sui_node/main.tf#L76-L83
https://www.notion.so/shinami/b5e1e9e16b7b49ffad58ca9cee09034b?v=77af7c369abb4ec0b57adc674009355e

**Review by ivanpeng-sh — APPROVED** [review]
